data/test-cases/plugins/gitlab/sidebar/Updated_LHS.md

```diff
@@ -0,0 +1,48 @@
+---
+# (Required) Ensure all values are filled up
+name: "LHS updates for the assigned MR, icons and tooltip names."
+status: Active
+priority: Normal
+folder: Sidebar
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Connect the Gitlab account to your MM account.
```

Comment:
Since we are logged in to MM, so we need to connect MM account ot Gutlab as all your actions would be performed on MM.